src/main/scala/TableGroup.scala

```diff
@@ -425,8 +431,7 @@ case class TableGroup private (
     id: Option[String],
     tables: Map[String, Table],
     notes: Option[JsonNode],
-    annotations: Map[String, JsonNode],
-    var warnings: Array[ErrorWithCsvContext]
+    annotations: Map[String, JsonNode]
```

Comment:
Do we ever actually use the `annotations` property?

Reply:
Maybe we can remove it, especially if the ruby code never uses it.

Reply:
We are classifying properties based on `annotations`, `common`, `inherited`. I dont really find any usages of this property, but if we have remove this annotations property from Table group, we might need to remove it from Table and Column too. Also rethink about this classification logic.

Reply:
Could you add a new issue to tidy this up later? We can ignore it for now.

Reply:
okay, here is the issue for it https://github.com/GSS-Cogs/csv-validation/issues/68